content/2016.12.give.a.new.chance.to.nightly.rst

```diff
@@ -0,0 +1,223 @@
+
+Anyone wants a stable Firefox and you don't have time for a crappy
+browser experience.
+
+Also, it was not always the case, Firefox Nightly is now much more
+stable than ever. The number of crashes has been schrinked down and it
+is getting lower really fast everyday.
+
+Also most of the bugs that you might have will be fixed the day after.
+
+You can always go in the `Help > About Firefox` menu to make sure you
+have the last update!
+
+
+Why you should actually give Firefox Nightly a try
+==================================================
+
+By installing Firefox Nightly you will:
+
+#. Have new Mozilla Firefox features 18 weeks before
+#. Have a much faster Firefox
+#. Make a difference
+#. Help reduce the number of crashes for Mozilla Firefox users
+#. Make sure Mozilla Firefox source code is in a workable state
+#. Help add-ons developer to validate that their add-ons is still working
+#. Have a better developer experience to develop your apps
+#. Give a really valuable contribution to the Mozilla project without coding
+#. Participate at building an Free Libre and Open Source browser that respect user privacy
+#. Have a real impact on how the Web is built noawadays.
+
+Have a much faster Firefox
+--------------------------
+
+Firefox is getting smaller new features are now tested through add-ons
+using `Test Pilot <https://testpilot.firefox.com/>`_ instead of
+getting added to Firefox.
+
+This means less bugs and only great features by default.
+
+Firefox is getting faster!
+
+This year 2017, Firefox is going to ship the `Quantum
+<https://wiki.mozilla.org/Quantum>`_ project which is a
+next-generation Web engine. It benefit from some critical parts of
+Gecko being rewritten in Rust which makes them 10 to 100 times faster
+than before by using parallelization and GPU offloading.
+
+This is happening now and you will get it 3 months ahead.
+
+Also Quantum is a lot more secure and the rewrite is a good
+opportunity to have a much recent and well tested code base which
+means that you are less likely to have bugs in it.
+
+Have new Mozilla Firefox features 18 weeks before
+-------------------------------------------------
+
+Mozilla Firefox is on a fixed Release Calendar workflow called trains,
+which means that developper can plan their feature to land in a
+certain version and coordinate other teams work (QA, OPS, Service
+developers, Deployments, Translations) for a certain date.
+
+In case the deadline is missed, they will have to wait for the next
+release 6 weeks later.
+
+Then there are 4 versions of Firefox:
+
+- Firefox Nightly, which is a tag of Mozilla Central every day
+- Firefox Developer edition, which is a tag of Nightly every 6 weeks
+- Firefox Beta, which is a release of the previous Developer edition branch
+- Firefox Release, which is a release of the previous Beta branch
+
+With Nightly, the release is everyday, which means you will be the
+first one to benefit from the new feature, while other people will
```

Comment:
Unless you're talking about a specific feature, `a new feature` (some generic hypothetical new feature) or `new features` (all of them). I prefer `new features`.